### PR TITLE
Feature/feature 4.2 design timeout handler 301

### PR DESCRIPTION
**Pull request by NicoDuvenage**
changed sequence diagrams to reflect required design in order to handle the TimeOuts

### CODE REVIEW

File: CentralServices/DA-Decisions/Decisions.txt [hunk 1]
```diff
@@ -0,0 +1,11 @@
+Decisions made by Design Authority regarding the story: 4.2 Design Timeout Handler #301
```
Comment (mdebarros) [inline CentralServices/DA-Decisions/Decisions.txt]: Please move this under CentralServices/designs
And rename the file using the following convention:
`des-<name>-<ref#>.md` <-- note that we only allow for MD files and not TXT files.  With regard to `ref#`, please use the main ref# for the related SDs (if it exists - which in this case is 2.3.0 I think).

File: CentralServices/seq_diagrams/seq-fulfil-2.1.0.plantuml [hunk 2]
```diff
@@ -124,7 +124,7 @@ group DFSP2 sends a Fulfil Success Transfer request
     FULF_HANDLER -> TOPIC_POSITION_DFSP2: Produce message
     |||
     TOPIC_POSITION_DFSP2 <- POS_HANDLER: Consume message
-    ref over TOPIC_POSITION_DFSP2, TOPIC_TRANSFERS:  Position Handler Consume (Success) {2.1.2.} \n
+    ref over TOPIC_POSITION_DFSP2, TOPIC_TRANSFERS:  Position Handler Consume (Success) {seq-position-1.3.2-fulfil} \n
```
Comment (mdebarros) [inline CentralServices/seq_diagrams/seq-fulfil-2.1.0.plantuml]: Why this change?

File: CentralServices/seq_diagrams/seq-fulfil-2.1.0.svg [hunk 3]
```diff
@@ -116,7 +116,7 @@ group DFSP2 sends a Fulfil Success Transfer request
     FULF_HANDLER -> TOPIC_POSITION_DFSP2: Produce message
     |||
     TOPIC_POSITION_DFSP2 <- POS_HANDLER: Consume message
-    ref over TOPIC_POSITION_DFSP2, TOPIC_TRANSFERS:  Position Handler Consume (Success) {2.1.2.} \n
+    ref over TOPIC_POSITION_DFSP2, TOPIC_TRANSFERS:  Position Handler Consume (Success) {seq-position-1.3.2-fulfil} \n
```
Comment (mdebarros) [inline CentralServices/seq_diagrams/seq-fulfil-2.1.0.svg]: Is this your change or is this coming from another fork?

File: CentralServices/seq_diagrams/seq-fulfil-2.1.0.plantuml (hunk 2 above)
Comment (mdebarros) [inline CentralServices/seq_diagrams/seq-fulfil-2.1.0.plantuml]: Is this your change or is this coming from another fork?

**Comment by shashi165** [comment]
raised another PR https://github.com/mojaloop/docs/pull/49
